Leaderboard/OfflineLeaderboardTab.cs

```diff
@@ -197,7 +197,7 @@ protected void OnDrawItem(GUI_OnlineLeaderboardPosition gui, IGetPersonalBests_R
         gui.pointsWon.gameObject.SetActive(_levelPoints.HasValue);
         if (_levelPoints.HasValue)
         {
-            gui.pointsWon.text = $"(+{(int)Math.Round(_levelPoints.Value * Math.Pow(0.995, index))})";
```

Comment:
Lmao did I mess it up again?

Reply:
No, it changed when making adjustments to the competitiveness metric a couple of weeks ago from the discussion in the various channels.